apps/public-reference/lib/ApplicationConductor.ts

```diff
@@ -62,13 +60,10 @@ export default class ApplicationConductor {
     }, 800)
   }
 
-  reset(shouldSync = true) {
+  reset() {
     this.application = blankApplication()
     this.listing = {} as Listing
-    if (shouldSync) {
-      this.context.syncApplication(this.application)
-      this.context.syncListing(this.listing)
```

Comment:
Are these two functions used after this refactor? If not, we should remove them from the context.

Reply:
@bencpeters They should probably be renamed—they're only relevant when actually replacing the entire JSON object with a new object. I don't think `syncApplication` is used at all right now, and `syncListing` is used only once when loading a listing from the API.